crates/oxc_linter/src/rules/import/no_namespace.rs

```diff
@@ -115,11 +115,9 @@ impl Rule for NoNamespace {
                             return;
                         }
 
-                        if self
-                            .ignore
-                            .iter()
-                            .any(|pattern| glob_match(pattern, source.trim_start_matches("./")))
-                        {
+                        if self.ignore.iter().any(|pattern| {
+                            glob_match(pattern.as_str(), source.trim_start_matches("./"))
```

Comment:
not sure if this is correct or whether CompactStr should impl AsRef<[u8]>

Reply:
Yes, you are correct. Since `CompactStr` implements the `Deref` trait, `&CompactStr` can be implicitly converted to `&str`. 

I had modified `fast-glob` to accept more generic input and didn’t anticipate this impact. Your changes are absolutely correct.